sidertests/test_types.py

```diff
@@ -112,3 +113,13 @@ def test_uuid(session):
                 b'\x11\xea.X\x97bG\xf3\xa31\xf2\xfaY\x95\xb7m', ByteString)
     with raises(ValueError):
         session.get(key('test_types_uuid'), UUID)
+
```

Comment:
Although it’s trivial style, please insert one more blank line.  According [PEP 8][1]:

>  Surround top-level function and class definitions with two blank lines.

[1]: https://www.python.org/dev/peps/pep-0008/#blank-lines